src/make_app.ts

```diff
@@ -49,7 +49,14 @@ export function makeApp(options: AppOptions): PolyserveApplication {
   app.get('*', function (req, res) {
     // Serve local files from . and other components from bower_components
     let url = parseUrl(req.url, true);
-    let splitPath = url.pathname.split('/').slice(1);
+    const pathPieces = url.pathname.split('/');
```

Comment:
I'd rather we use `let` for locals unless we convert the entire project to `const`, for consistency.